transports/msmq/index.md

```diff
@@ -2,16 +2,42 @@
 title: MSMQ Transport
 summary: MSMQ is the primary durable communications technology for Microsoft but does not dynamically detect network interfaces.
 component: MsmqTransport
+related:
+ - samples/msmq/simple
+ - samples/msmq/persistence
+ - samples/msmq/dlq-customcheck
+ - samples/msmq/msmqtosqlrelay
 reviewed: 2016-08-31
 redirects:
  - nservicebus/msmq-information
  - nservicebus/msmq
+tags:
+ - Transport
 ---
 
+## Configuring the endpoint
+
 partial: default
 
 
-## NServiceBus Configuration
+## Advantages and Disadvantages
+
+
+### Advantages
+
+ * MSMQ is natively available as part of the Windows operating system. In Servers, the MSMQ role might need to be turned on.  
+ * MSMQ offers transactional queues which also support distributed transactions. With the transactional behavior, it is easy to get once-only delivery achievable.
+ * MSMQ provides a store and forward mechanism. Therefore it promotes a more natural bus-style architecture. When sending messages to unavailable servers, the messages are stored in the Outgoing queues and will be automatically delivered once the machine comes back online.
```

Comment:
Link to https://docs.particular.net/transports/#types-of-transports-bus-transports ?